deviceComunicationDocumentation.MD

```diff
@@ -956,25 +885,22 @@ Defaultní hodnota je 12 hodin od poslední odeslané zprávy.
 | 2.byte | Minuty                         |
 | 3.byte | Sekundy                        |
 
-#### Nasavení jak často se má odeslat measure zpráva
+#### Nasavení, jak často se má odeslat measure zpráva
 
 Popsáno zde (TODO odkaz)
 
```

Comment:
Kam má tohle odkazovat? Není to celé zbytečné? Popis je pod tím ...

Reply:
pridano